src/node/config.ts

```diff
@@ -219,6 +219,13 @@ export interface HmrConfig {
   hostname?: string
   port?: number
   path?: string
+  /**
+   * If you are using hmr ws proxy, it maybe timeout with your proxy program.
+   * You can set this option to let client send ping socket to keep connection alive.
+   * The option use `second` as unit.
```

Comment:
Why "seconds"? I think it's more intuitive to use milliseconds as other built-in functions also use them (e.g. `setTimeout`)